### ISSUE TITLE
`MD029 Ordered list item prefix` Triggered

### ISSUE THREAD

**Issue by saimsalman36** [markdownlint/markdownlint#268]
I have a README.md file with:

1. `normal`
2. `lognormal`
3. `beta`
4. `chi-squared`
5. `exp`
6. `f`
7. `gamma`
8. `gumbel-right`
9. `inverse-gamma`
10. `laplace`
11. `pareto`
12. `studentst`
13. `weibull`

but the following warnings are triggered

`README.md:172: MD029 Ordered list item prefix`

for each line.

If I change all the numbers to `1` then it works.

**Comment by bombsimon** [comment]
The number in an ordered list has no meaning, see [basic syntax](https://www.markdownguide.org/basic-syntax/#ordered-lists) for ordered list. One of the benefits of using the same number (number 1 since it has to start with that number) is that you can add any element anywhere in the list without having to change the numbers for the rest of the items. Imagine you would like to add an item at position 3 in your list above, it would be a hassle to change 4-13.

If you want to use different numbers maybe it's better to disable `MD029` instead.

**Comment by DavidAnson** [comment]
FYI: The Node implementation of this rule supports both styles. It may be worth considering porting that over to the Ruby implementation.

https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md029---ordered-list-item-prefix

**Comment by jaymzh** [comment]
this rule supports both styles as well:

```
Parameters: style ("one", "ordered"; default "one")
```

If you prefer this style, then please configure your rule to `ordered` instead of the default `one`.
